Automattic/wp-calypso
# Remove unused WpcomFAQ component

Related to https://github.com/Automattic/wp-calypso/pull/78262#issuecomment-1593092943

## Proposed Changes

* Removes the WpcomFAQ component as it is not reachable by any code path anymore.

## Testing Instructions



* Do a code search and confirm that this is indeed dead code. 
* Although E2E tests should take care of this, spot check a few flows: `/start`, `/plans/<site slug>`, `/setup/newsletter/plans`.

## Pre-merge Checklist
